src/main/java/org/prgrms/cream/domain/deal/service/SellingService.java

```diff
@@ -0,0 +1,61 @@
+package org.prgrms.cream.domain.deal.service;
+
+import java.time.format.DateTimeFormatter;
+import org.prgrms.cream.domain.deal.domain.SellingBid;
+import org.prgrms.cream.domain.deal.dto.BidRequest;
+import org.prgrms.cream.domain.deal.dto.BidResponse;
+import org.prgrms.cream.domain.deal.repository.SellingRepository;
+import org.prgrms.cream.domain.product.domain.ProductOption;
+import org.prgrms.cream.domain.product.service.ProductService;
+import org.prgrms.cream.domain.user.exception.InvalidArgumentException;
+import org.prgrms.cream.domain.user.service.UserService;
+import org.prgrms.cream.global.error.ErrorCode;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+@Service
+public class SellingService {
+
+	private final SellingRepository sellingRepository;
+	private final ProductService productService;
+	private final UserService userService;
+
+	public SellingService(
+		SellingRepository sellingRepository,
+		ProductService productService,
+		UserService userService
+	) {
+		this.sellingRepository = sellingRepository;
+		this.productService = productService;
+		this.userService = userService;
+	}
+
+	@Transactional
+	public BidResponse registerBuyingBid(Long id, String size, BidRequest bidRequest) {
+		ProductOption productOption = productService.findProductOptionByProductIdAndSize(id, size);
+
+		if (bidRequest.price() < productOption.getBuyLowestPrice()) {
```

Comment:
```suggestion
		if (bidRequest.price() < productOption.getBuyLowestPrice() || productionOption.getBuyLowestPrice() == 0) {
```

productOption의 해당 필드가 default 값(0)으로 저장되어 있을수도 있어서 최저가 갱신을 위해서는 하나의 조건을 더 달아줘야 할 것 같습니다!

Reply:
이 부분도 위와 마찬가지로 생각해봤는데 조건에 대한 문제가 뭔지 이해했습니다! ㅎㅎ 수정하겠습니다!